questionpy_sdk/webserver/question_ui/__init__.py

```diff
@@ -267,10 +273,8 @@ def _replace_qpy_urls(self, xml: str) -> str:
     def _resolve_placeholders(self) -> None:
         """Replace placeholder PIs such as `<?p my_key plain?>` with the appropriate value from `self.placeholders`.
 
-        TODD: remove comment or change call-order
-
         Since QPy transformations should not be applied to the content of the placeholders, this method should be called
-        last.
+        near the end (see `_clean_up` method).
```

Comment:
```suggestion
        near the end (after `_clean_up`)
```

Vgl. im Plugin: `near the end (after {@see clean_up()}).`